packages/dapp/src/components/StatsCard/StatsCard.tsx

```diff
@@ -117,6 +121,16 @@ export const StatsCards = ({
       open: openChainsPopper,
       setOpen: setOpenChainsPopper,
       handleOnClick: () => {
+        trackEvent({
+          category: TrackingCategory.WelcomeScreen,
+          action: TrackingAction.OpenStatsModal,
+          label: 'chains_stats',
+          data: { [TrackingEventParameter.StatsModal]: 'chains_stats' },
```

Comment:
"`chains_stats`" is it a defined type or raw string?

Reply:
We have typed the keys but not the values. The idea was that these may be used as Custom Dimensions / Metrics in Google-Analytics.

So, that way "chains_stats" is a raw string.